MovieApp/MovieApp/App/TabBar/CustomTabBarController.swift

```diff
@@ -23,17 +23,25 @@ class CustomTabBarController: UITabBarController {
         
         buildViews()
         setupChildViewControllers()
+        setupNavigationViewController()
     }
     
     private func setupChildViewControllers() {
         addChild(homeViewController)
-        addChild(favoriteMoviesViewController)
+        addChild(favoriteViewController)
         
         homeViewController.tabBarItem.image = UIImage(with: .homeLogo)
         homeViewController.tabBarItem.title = LocalizableStrings.tabBarHomeTitle.rawValue
         
-        favoriteMoviesViewController.tabBarItem.image = UIImage(with: .favoriteLogo)
-        favoriteMoviesViewController.tabBarItem.title = LocalizableStrings.tabBarFavoritesTitle.rawValue
+        favoriteViewController.tabBarItem.image = UIImage(with: .favoriteLogo)
+        favoriteViewController.tabBarItem.title = LocalizableStrings.tabBarFavoritesTitle.rawValue
     }
     
+    
+    private func setupNavigationViewController() {
+        let logo = UIImage(with: .appLogo)
+        let logoImageView = UIImageView()
+        logoImageView.image = logo
+        navigationItem.titleView = logoImageView
+    }
```

Comment:
check documentation about navigationControllers. This is completely different thing from viewControllers so you will never be in doubt again

Reply:
change naming, UIViewControllers and UINavigationController are no way near similar

Reply:
and you are not setting your navController here, you are just adding navItem to the navController